docs/spark/how-to-guides/batch-processing.md

```diff
@@ -0,0 +1,124 @@
+---
+title: How to do batch processing with .NET for Apache Spark
+description: Learn how to do batch processing using .NET for Apache Spark.
+ms.date: 11/14/2019
+ms.topic: conceptual
+ms.custom: mvc,how-to
+---
+
+# How to do batch processing with .NET for Apache Spark
+
+This how-to shows you how to do batch processing using .NET for Apache Spark. If this is your first time using .NET for Apache Spark, check out the [Get started with .NET for Apache Spark](../tutorials/get-started.md) tutorial to learn how to prepare your environment and run your first .NET for Apache Spark application.
+
+## What is batch processing?
+
+Batch processing is the transformation of data at rest, meaning that the source data has already been loaded into data storage. Batch processing is generally performed over large, flat datasets that need to be prepared for further analysis. Log processing and data warehousing are common batch processing scenarios. 
+
+## GitHub projects data example
+
+The following steps demonstrate batch processing of GitHub projects data. 
+
+[GHTorrent](http://ghtorrent.org/) monitors all public GitHub events, such as info about projects, commits, and watchers, and stores the events and their structure in databases. Data collected over different time periods is available as downloadable archives. Because the dump files are very large, this guide uses a [truncated version of the dump file]() that can be downloaded from GitHub.
```

Comment:
Once [this PR](https://github.com/dotnet/spark/pull/322) gets merged, the final URL for the dataset should be: `https://github.com/dotnet/spark/tree/master/examples/Microsoft.Spark.CSharp.Examples/Sql/Batch/projects_smaller.csv`